packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -87,72 +87,69 @@ import './colors.css';
 ## Status and state colors
 <Grid hasGutter>
   <GridItem xl={6} lg={12} className="ws-colors-gridtext">
-    Status and state colors are important when communicating data and action aspects through the UI with the user. Our status colors cover a variety of statuses including default, danger, success, information, and warning.
+    Status and state colors are important considerations when communicating data and actions to the user through the UI. Our status colors cover default, danger, success, information, and warning statuses.
```

Comment:
Made that change and also added "disabled" here. It seems to me like "disabled" refers to a "state", rather than a "status", but we don't have specific guidance on that distinction, so please let me know if it's typically used in a different way!

Reply:
I think that's a good distinction between "state" and "status."